index.bs

```diff
@@ -2739,7 +2773,7 @@ attributes.
 
         The [=[RP]=] can determine the resulting [=authenticator attachment modality|attachment=] for the created credential
         using the {{AuthenticatorAttestationResponse/getTransports()}} method of the resulting {{AuthenticatorAttestationResponse}}.
-        See [[#enum-transport]] for details.
+        See [[#sctn-attachments-from-transports]] for details.
```

Comment:
I like the new section, but as stated in the review of #1621 I am not sure if it's deterministically possible to derive the authenticator attachment modality that was used purely from the result of `getTransports()`, specifically in the case where `getTransports()` returns more than one value, and one of the values is `internal`. The RP cannot determine which was actually used, and therefore cannot determine whether or not it is useful to prompt the end user to register the platform authenticator if one exists. Let's discuss in the WebAuthn WG call, but I'm thinking the proposal in #1168 would satisfy this goal deterministically.

Reply:
and delete this sentence
```suggestion
```